sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -274,7 +293,12 @@ def get_custom_model(self, model_id, **kwargs):
             raise ValueError("model_id cannot be None or empty.")
 
         response = self._client.get_custom_model(model_id=model_id, include_keys=True, error_map=error_map, **kwargs)
-        return CustomFormModel._from_generated(response)
+        try:
```

Comment:
same here, and below - error handling should surround the client generated code call